assets/RFC003-EthereumBitcoin-AtomicSwap.puml

```diff
@@ -0,0 +1,132 @@
+' DIAGRAM #########################################
+' RFC003 atomic swap
+' #################################################
+@startuml
+' #################################################
+' SETTINGS: color settings of diagram
+' #################################################
+skinparam sequence {
+	BorderColor black
+	ArrowColor black
+	ActorBorderColor black
+	LifeLineBorderColor black
+	LifeLineBackgroundColor white
+	
+	ParticipantBorderColor black
+	ParticipantBackgroundColor white
+	ParticipantFontColor black
+	
+	ActorBackgroundColor white
+	ActorFontColor black
+}
+' #################################################
+' ACTORS
+' #################################################
+actor Alice
+participant "Alice\nComit-Node" as AC #lightgreen
+participant "Alice\nETH-Client" as AETH #lightcyan
+participant "Alice\nBTC-Client" as ABTC #lightgoldenrodyellow
+participant "Ethereum\nNetwork" as ETH #lightcyan
+participant "Bitcoin\nNetwork" as BTC #lightgoldenrodyellow
+participant "Bob\nBTC-Client" as BBTC #lightgoldenrodyellow
+participant "Bob\nETH-Client" as BETH #lightcyan
+participant "Bob\nComit-Node" as BC #lightgreen
+actor Bob
+
+
+' #################################################
+' DIAGRAM
+' #################################################
+Bob->BC: create comit link
+BC->Bob: link for trade
+Bob->Alice: comit link, 10ETH to 1BTC
+note left #white
+	Trade is set up so:
+	Alice has 10 ETH 
+	  and gets 1 BTC.
+	Bob has 1BTC 
+	  and gets 10 ETH.
+end note
+Alice->AC: refund address
+Alice->AC: send swap request
+AC->AC: generate secret
+AC->AC: hash secret, h(s)
+note left #white
```

Comment:
🙃  it is redundant from the 2 lines above. Either keep the notes or the `AC->AC:...` not both.

Reply:
fixed.